src/python/pants/source/source_root.py

```diff
@@ -255,58 +251,51 @@ def register_options(cls, register):
         )
 
         register(
-            "--root-patterns",
-            metavar="<list>",
+            "--roots",
+            metavar='["path/to/root1", "path/to/root2", ...]',
             type=list,
             fingerprint=True,
             # For Python a good default might be the repo root, but that would be a bad default
             # for other languages, so best to have no default for now, and force users to be
             # explicit about this when integrating Pants.  It's a fairly trivial thing to do.
             default=[],
             advanced=True,
-            help="A list of source root suffixes. A directory with this suffix will be considered "
-            "a potential source root. E.g., `src/python` will match `<buildroot>/src/python`, "
-            "`<buildroot>/project1/src/python` etc. Prepend a `^/` to anchor the match at the "
-            "buildroot.  E.g., `^/src/python` will match `<buildroot>/src/python` but not "
-            "`<buildroot>/project1/src/python`.  A `*` wildcard will match a single path segment, "
-            "e.g., `src/*` will match `<buildroot>/src/python` and `<buildroot>/src/rust`. "
+            help="A list of source roots, relative to the repo root. A source root represents "
```

Comment:
In documentation, I've been using `repository`. It removes any possible ambiguity and is more searchable.

```suggestion
            help="A list of source roots, relative to the repository root. A source root represents "
```